sbtc/tests/integration/rpc.rs

```diff
@@ -0,0 +1,88 @@
+//! Test the RPC clients
+
+use bitcoin::hashes::Hash;
+use bitcoin::AddressType;
+use sbtc::error::Error;
+use sbtc::testing::regtest;
+use sbtc::testing::regtest::Recipient;
+
+use sbtc::rpc::BitcoinRpcClient;
+use sbtc::rpc::BtcClient;
+use sbtc::rpc::ElectrumClient;
+use test_case::test_case;
+
+#[cfg_attr(not(feature = "integration-tests"), ignore)]
+#[test_case(BtcClient::new(
+        "http://localhost:18443",
```

Comment:
Should we use a hardcoded port? Wonder if it would be better to select one dynamically? Same goes with below.